operator/validator/controller.go

```diff
@@ -517,12 +521,22 @@ func (c *controller) StartValidators() {
 	}
 
 	var ownShares []*ssvtypes.SSVShare
-	var allPubKeys = make([][]byte, 0, len(shares))
+	var pubKeysToFetch [][]byte
 	for _, share := range shares {
 		if c.operatorDataStore.GetOperatorID() != 0 && share.BelongsToOperator(c.operatorDataStore.GetOperatorID()) {
 			ownShares = append(ownShares, share)
 		}
-		allPubKeys = append(allPubKeys, share.ValidatorPubKey[:])
+		committeeSubnet := byte(commons.CommitteeSubnet(share.CommitteeID()))
+		if bytes.Contains(c.network.ActiveSubnets(), []byte{committeeSubnet}) {
+			pubKeysToFetch = append(pubKeysToFetch, share.ValidatorPubKey[:])
+		}
+
+		if !c.networkConfig.PastAlanFork() {
```

Comment:
since this version is going to be released after mainnet fork, there's no reason to include fork code really, we're planning to remove all pre-fork logic in this release anyway.